Sources/MapboxNavigation/UserHaloCourseView.swift

```diff
@@ -7,21 +7,6 @@ import CoreLocation
 public class UserHaloCourseView: UIView, CourseUpdatable {
     private var lastLocationUpdate: Date?
 
-    /**
-     Transforms the location of the user halo.
-     */
-    public func update(location: CLLocation, pitch: CGFloat, direction: CLLocationDegrees, animated: Bool, tracksUserCourse: Bool) {
```

Comment:
This implementation is similar to default one in `CourseUpdatable`, so can be removed.